Server/src/services/queries.js

```diff
@@ -0,0 +1,16 @@
+import { Pool } from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const pool = new Pool({ connectionString: process.env.DATABASE_URL });
```

Comment:
are you really using pool in this file? It seems unused since your never calling it or exporting it